src/components/detailedView/instanceView/instanceView.tsx

```diff
@@ -30,38 +30,34 @@ async function getData(instancekey: string, token: string) {
 }
 
 interface InstanceViewState {
-  state: InstanceObject | undefined,
-  columns: Array<string>,
-  fields: Array<string>
+  state: InstanceObject | undefined;
+  columns: Array<string>;
+  fields: Array<string>;
 }
 
 export class InstanceView extends React.PureComponent<
   RouteComponentProps & InstanceViewProps,
   InstanceViewState
 > {
-
   public state: InstanceViewState = {
     state: undefined,
     columns: ["Hostname", "Model", "Rack", "Elevation", "Owner"],
     fields: ["hostname", "model", "rack", "elevation", "owner"]
-  }
+  };
 
   public render() {
     let params: any;
     params = this.props.match.params;
-    if (this.state.state === undefined){
-      getData(params.rid, this.props.token).then((result) => {
+    if (this.state.state === undefined) {
+      getData(params.rid, this.props.token).then(result => {
         this.setState({
           state: result
-        })
-      })
+        });
+      });
     }
     return (
       <div className={Classes.DARK + " instance-view"}>
-        <PropertiesView
-          history={this.props.history} location={this.props.location}
-          match={this.props.match} data={this.state.state} {...this.state}
-        />
+        <PropertiesView data={this.state.state} {...this.state} />
```

Comment:
you weren't wrapping PropertiesView with withRouter(), which is why you had to pass in all these props manually